workers/redirector.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * This is a custom URL redirector implemented as a CloudFlare Worker.
+ * The role of the worker is to redirect a predefined % of traffic 
+ * to a different URL (used for A/B testing).
+ * API documentation can be found at https://developers.cloudflare.com/workers/
+ */
+
+/**
+ * The origin that the worker is to be run on.
+ * (e.g. https://www.allizom.org, https://www.mozilla.org).
+ */
+const origin = 'https://www-dev.allizom.org';
```

Comment:
As discussed, I think it would be better to remove the hard-coded origin so that we can use the same script on any domain.